### PR TITLE
LPS-198159 | Poshi Automation | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# LPS-177756 - Product QA | Test Automation Creation

Hi! :)
This pr is corresponding to this ticket: [LPS-177756](https://liferay.atlassian.net/browse/LPS-177756) → [LPS-198159](https://liferay.atlassian.net/browse/LPS-198159)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -6592,6 +6592,74 @@ definition {
...
 
+	@description = "This is a use case for LPS-177756. The tooltip of the back button is Go to Content Dashboard when viewing an asset with a DPT from the Content Dashboard."
+	@priority = 3
+	test ViewBackButtonTooltipInContentDashboard {
+		task ("Given the Document edit mode page") {
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = "DM Document Title",
+				groupName = "Test Site Name",
+				mimeType = "text/txt",
+				sourceFileName = "Document_1.txt");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:6603]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 2]
```diff
@@ -6592,6 +6592,74 @@ definition {
...
+
+			JSONWebcontent.addWebContent(
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title");
+
+			JSONBlog.addEntry(
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title",
+				groupName = "Test Site Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:6613]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 3]
```diff
@@ -6592,6 +6592,74 @@ definition {
...
+	@description = "This is a use case for LPS-177756. The tooltip of the back button is Go to Content Dashboard when viewing an asset with a DPT from the Content Dashboard."
+	@priority = 3
+	test ViewBackButtonTooltipInContentDashboard {
+		task ("Given the Document edit mode page") {
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = "DM Document Title",
+				groupName = "Test Site Name",
+				mimeType = "text/txt",
+				sourceFileName = "Document_1.txt");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:6604]: Add steps to add a display page template for Web Content Article and Basic Web Content then associate display page to web content

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 4]
```diff
@@ -6592,6 +6592,74 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-177756. The tooltip of the back button is Go to Content Dashboard when viewing an asset with a DPT from the Content Dashboard."
+	@priority = 3
+	test ViewBackButtonTooltipInContentDashboard {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:6597]: ViewBackButtonTooltipInAssociatedDisplayPageTemplate

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardWithKeyboard.testcase [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+		User.firstLoginPG();
+
+		HeadlessSite.addSite(siteName = "Test Site Name");
+
+		Category.openCategoriesAdmin(siteURLKey = "test-site-name");
+	}
+
+	@description = "This is a test for LPS-177756. Using Keyboard Navigation it is possible to access the back button of the DPTs open from Content Dashboard and the tooltip of the back button is Go to Content Dashboard."
+	@priority = 3
+	test ViewBackButtonTooltipInContentDashboardViaKeyboard {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardWithKeyboard.testcase:23]: ViewBackButtonTooltipInAssociatedDisplayPageTemplateViaKeyboard

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardWithKeyboard.testcase [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+			JSONWebcontent.addWebContent(
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title");
+
+			JSONBlog.addEntry(
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title",
+				groupName = "Test Site Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardWithKeyboard.testcase:39]: Same with above

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 7]
```diff
@@ -6592,6 +6592,74 @@ definition {
...
+
+		task ("When hover over the back button") {
+			MouseOver(locator1 = "Icon#BACK");
+		}
+
+		task ("Then see the tooltip message") {
+			AssertVisible(
+				key_content = "Go to Content Dashboard",
+				locator1 = "Tooltip#FLOATING_TOOLTIP_CONTENT");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:6628]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 8]
```diff
@@ -6592,6 +6592,74 @@ definition {
...
+	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:6661]: No need

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @rafaellajordao, would you mind updating according to comments? Thanks

**Comment by Tim-Cao** [comment]
@rafaellajordao Good job! See changes https://github.com/liferay-echo/liferay-portal/pull/14891

**Pull request closed without merging**